src/api/Flooq.Api/Controllers/VersionController.cs

```diff
@@ -1,15 +1,30 @@
 using Microsoft.AspNetCore.Mvc;
+using Flooq.Services;
 
 namespace Flooq.Controllers
 {
-  [Route("api/[controller]")]
+  [Route("api/version")]
```

Comment:
I'm ok that we override the default `api/Version` to `api/version` but we just have to keep in mind to do this on all controllers.